### ISSUE TITLE
Change Safety of bases

### ISSUE THREAD

**Issue by LukyLucas** [MineInAbyss/Issue-Tracker#116]
### Is your feature request related to a problem?

Alot of people on the server would be against this however I think the ability to protect blocks in pvp layers should be removed. BUT HEAR ME OUT, the ability to raid bases in a pvp layer would be a interesting change to test in the abyss. Our current player base is extremely peaceful to the point we can build cities and towns on the server. Most of this i believe stems from the point that keep inventory is on and griefing/raiding is grounds for a ban. I say we should try removing inventory and base protection on Any layer with pvp on. The areas above the goblets would be the same but with the newfound danger of other players in the abyss people would be forced to expand on pvp which is a very rare thing in the abyss currently. Players would hide thier bases or community's would have to protect thier land wherever players can fight. Another aspect is with the new addition of those who try to raid others is the fact that they will always try to have the upper hand, which means exploiters. those who find holes in the system for thier own benefit. We would slowly find more and more things to patch which will just improve our server and expand the player population. 

### Describe the solution you'd like.

This will probably be rejected and anyone with bases below in l4/5 would be pissed however in terms of looking at the future of the server I see nothing but benefits. 

### Describe alternatives you've considered.

Basically layers 1-3 y the same however layers 4 and below loose raid protection. 

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest supported Paper build.

**Comment by LukyLucas** [comment]
I imagine this will be very controversial

**Comment by Boy0000** [comment]
This will not happen no.

**Comment by LukyLucas** [comment]
Is there a specific reason? I see nothing but upsides

**Comment by Boy0000** [comment]
You literally say it in your idea. A vast majority of players would hate it.
Survival is not heavily focused on PVP and therefore adding feature that encourage just toxic behaviour between players is not very pog. All it means is that people will hide their stuff inside random walls and thus ESPs become a lot more used.

I see nothing but downsides

**Comment by LukyLucas** [comment]
It's only in pvp layers and it isn't totally bad, the player base would expand significantly and the staff for the server may also expand in order to deal with new branch of players in the community. May i offer the idea of testing it by backing up the current save, and applying the suggested changes for a limited time or on a duplicate server? The peaceful community would stick to higher layers and pvp would just make dying or making the journey so much more dangerous. Which is good.

**Comment by LukyLucas** [comment]
In the anime the biggest danger was the possibility of running into other delvers

**Comment by Boy0000** [comment]
The feature request is closed, answer is no it wont be implemented for the above-mentioned reasons

**Comment by LukyLucas** [comment]
Can this at least be added to a thread in suggestions and have player vote over this with a list of possible benefits and losses? This would be game changing and massive in the servers future.

**Comment by LukyLucas** [comment]
Only those with large bases below l3 would be super angry though

**Comment by Boy0000** [comment]
![image](https://user-images.githubusercontent.com/62521371/163598390-eb7adac1-3774-4ad7-9e55-48a87d8b99f0.png)

**Comment by LukyLucas** [comment]
Fine

**Comment by LukyLucas** [comment]
Sorry not trying to be annoying or push right before the end of my mute but could you atleast bring up the topic during a staff meeting. If love to see how this changes the servers future. Sorry again

**Comment by Boy0000** [comment]
![image](https://user-images.githubusercontent.com/62521371/163598936-207d3e98-042c-4ad6-a2c4-fc2d61d2ec96.png)

**Comment by LukyLucas** [comment]
Ok...
